plugin-modernizer-cli/src/main/java/io/jenkins/tools/pluginmodernizer/cli/Main.java

```diff
@@ -37,6 +37,9 @@ public static void main(final String[] args) {
     @Option(names = {"-r", "--recipes"}, required = true, description = "List of Recipes to be applied.", split = ",", parameterConsumer = CommaSeparatedParameterConsumer.class)
     private List<String> recipes;
 
+    @Option(names = {"-g", "--github-owner"}, description = "GitHub owner for forked repositories (only username supported for now)")
+    private String githubUsername = Settings.GITHUB_USERNAME;
```

Comment:
To be renamed to githubOwner when we support also organisation.on the `GHService` we should be able to do it transparently if forking to user or an other organisation